package.json

```diff
@@ -18,10 +18,10 @@
     "dist/fusionauth-typescript-client.*"
   ],
   "scripts": {
-    "test": "mocha build/test/FusionAuthClientTest.js && karma start",
-    "build-browser": "mkdir -p dist && npx browserify index.ts --standalone FusionAuth --debug -p [ tsify --target=es5 ] -o dist/fusionauth-typescript-client.js",
-    "build-browser-min": "mkdir -p dist && npx browserify index.ts --standalone FusionAuth --debug -p [ tsify --target=es5 ] -t uglifyify | npx uglifyjs --source-map base -o dist/fusionauth-typescript-client.min.js",
-    "pretest": "npx tsc && mkdir -p dist && npx browserify test/FusionAuthClientTest.ts --debug -t envify -p [ tsify --target=es5 ] -t uglifyify | npx uglifyjs --source-map base -o dist/fusionauth-typescript-client-test.min.js",
+    "test": "npx mocha build/test/FusionAuthClientTest.js && npx karma start",
```

Comment:
A little safer on platforms that might pull our repo and not have mocha or karma globally